### DISCUSSION TITLE
Scripts link (MS SQL and Oracle)

### DISCUSSION THREAD

**Discussion by Sergic271** [dbeaver/dbeaver#34457] [Q&A]
Hi everyone,

I have my DBeaver connected to MS SQL and Oracle databases, both have certain scripts which I use daily.
Is there any possibility to chain link MS SQL script and Oracle script execution and pass a value from one to another?

For example, I make certain changes in MS SQL database using script which uses "RegNumber" field value.
After that I have to run Oracle script with the same "RegNumber" field value - but in different database (Oracle).

Can I somehow pass the value of 'RegNumber' from MS SQL script ot Oracle script in DBeaver?

**Comment by E1izabeth** [comment]
As far as I know, it's not possible

**Comment by TB-HunBoka** [comment]
Based on your example if you already know the "RegNumber" that you would like to use, you can create two database tasks and assign this value as a variable for them and running those tasks after each other.

Query 1 at mysql:
```
update something in mysql.table
where id = :test_regnum
```

Query 2 at oracle:
```
Update something in oracle.table
where row_id = :test_regnum
```
Would that be what you want?

If you would like to get the RegNumber from my sql and run a query in oracle based on the result, then I do not know how you could do that in dbeaver.
You can achive this by connecting the MS SQL to your Oracle server or develop it, for example in python.
